presentation/src/main/java/com/depromeet/presentation/viewfinder/dialog/StadiumSelectSeatDialog.kt

```diff
@@ -133,6 +132,19 @@ class StadiumSelectSeatDialog : BindingBottomSheetDialog<FragmentStadiumSelectSe
         }
     }
 
+    private fun handleEditTextFocus() {
+        binding.etNumber.setOnFocusChangeListener { v, hasFocus ->
+            if (!hasFocus) {
+                binding.ivNumberCancel.visibility = View.INVISIBLE
```

Comment:
저번에 달려다가 말았던 부분인데 요렇게 바꾸면 더 가독성이 좋아지는 것 같아요 ! 

```suggestion
                binding.ivNumberCancel.visibility = INVISIBLE
```

Reply:
이거는 이제부터 이렇게 해볼게! 지금까지 모든 Visible 처리를 View참조해서 했기 때문에 다 찾기가 힘들듯.. 🫣